impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -212,14 +240,8 @@ std::optional<Entity> GaussianBlurFilterContents::RenderFilter(
                                 entity.GetClipDepth());  // No blur to render.
   }
 
-  Scalar blur_radius = CalculateBlurRadius(sigma_);
   Scalar desired_scalar = CalculateScale(sigma_);
-  // TODO(jonahwilliams): if scaling value is 1.0, then skip the downsample
-  // pass.
-
```

Comment:
We can't skip downsampling because it downsampling also inserts the gutter required for the halo.